app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  after_create :update_posts_counter
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  def update_posts_counter
+    User.find_by(name: author.name).increment!(:posts_counter)
+  end
+
```

Comment:
- [ ] Great job making use of [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html). But when ever a `callback` is used, the method that uses it is made `private`. Kindly check the example below using the callback `after_save` and fix your code accordingly. Also take the `latest_comments` method to `lines 7 to 9` and take your private method below.
 ```
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end

```